subprojects/docs/src/main/resources/header.html

```diff
@@ -181,15 +181,22 @@ <h3 id="authoring-gradle-builds">Authoring Gradle Builds</h3>
                     <li><a href="../userguide/potential_traps.html">Avoiding Traps</a></li>
                 </ul>
             </li>
-            <li><a class="nav-dropdown" data-toggle="collapse" href="#authoring-multi-project-builds" aria-expanded="false" aria-controls="authoring-multi-project-builds">Authoring Multi-Project Builds</a>
-                <ul id="authoring-multi-project-builds">
+            <li><a class="nav-dropdown" data-toggle="collapse" href="#authoring-multi-project-builds" aria-expanded="false" aria-controls="authoring-multi-project-builds">Structuring a S/W Component</a>
```

Comment:
I think this sounds a bit abstract.  WDYT of...

- Organizing Gradle builds
- Scaling up Gradle builds

Reply:
Mh. I am also not super happy with this. But I also don't like the suggestions a lot.

"Organizing Gradle builds" applies to it all
I don't like "Upscaling" very much. It sounds like something you would always do later to me. While you can very well decide to use a structure that is ready to reflect your architecture from the beginning on.

How about something in the direction of:

* Structuring Individual Builds
* Structuring Software Product Projects